src/lib/db-listeners/db-other-docs-listener.ts

```diff
@@ -34,11 +35,17 @@ export class DBOtherDocumentsListener extends BaseListener {
       this.publicationsPath = this.db.firebase.getClassPublicationsPath(this.db.stores.user);
     }
 
-    this.documentsRef = this.db.firebase.ref(this.documentsPath);
-    this.debugLogHandlers("#start", "adding", ["child_added", "child_changed", "child_removed"], this.documentsRef);
-    this.documentsRef.on("child_added", this.handleDocumentAdded);
-    this.documentsRef.on("child_changed", this.handleDocumentChanged);
-    this.documentsRef.on("child_removed", this.handleDocumentRemoved);
+    const documentsRef = this.db.firebase.ref(this.documentsPath);
+    this.documentsRef = documentsRef;
+    documentsRef.once("value", snapshot => {
+      if (size(snapshot.val()) === 0) {
+        this.db.stores.documents.resolveRequiredDocumentPromise(null, this.documentType);
+      }
```

Comment:
We add the `once` handler so that we can tell when there are no documents, which wouldn't otherwise trigger one of our handlers.